crates/holochain_cascade/CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## \[Unreleased\]
 
+- The feature flag `unstable-warrants` must be enabled for `handle_get_agent_activity` to query for warrants. If the feature is disabled the returned list of warrants will always be empty.
+- The feature flag `unstable-warrants` must be enabled for `get_bounded_activity` to query for warrants. If the feature is disabled the returned list of warrants will always be empty.
```

Comment:
```suggestion
- The feature flag `unstable-warrants` must be enabled for `handle_get_agent_activity` to query for warrants. If the feature is disabled, the returned list of warrants will always be empty.
- The feature flag `unstable-warrants` must be enabled for `get_bounded_activity` to query for warrants. If the feature is disabled, the returned list of warrants will always be empty.
```

Reply:
Done, but moved to `holochain/CHANGELOG`: https://github.com/holochain/holochain/pull/5319/commits/7bf515c422a8a052d08854a64bc073e852fbb9de